storage/innobase/xtrabackup/doc/source/xtrabackup_bin/xbk_option_reference.rst

```diff
@@ -657,4 +658,10 @@ Options
 
    This option prints |xtrabackup| version and exits.
 
+.. option:: --xtrabackup-plugin-dir=DIRNAME
+
+   The absolute path of the directory that contains the
+   ``keyring_file`` plugin (:file:`keyring_file.so`). This option is
```

Comment:
Not just keyring_file. xtrabackup shipped with two plugins: keyring_file and keyring_vault. I would not mention keyring-file-data specifically, instead I would give a link to https://www.percona.com/doc/percona-server/LATEST/management/data_at_rest_encryption.html#keyring-vault-plugin and https://dev.mysql.com/doc/refman/5.7/en/keyring-file-plugin.html for option reference

Reply:
Ready for review